dashboard/test/helpers/pd/workshop_survey_results_helper_test.rb

```diff
@@ -39,108 +39,31 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
-    create :pd_enrollment, workshop: workshops.first
-
-    result_hash = summarize_workshop_surveys(workshops: workshops)
-    assert_equal 4, result_hash[:num_enrollments]
-    assert_equal 3, result_hash[:num_surveys]
-    assert_equal 4, result_hash[:how_much_learned]
-    assert_equal ['venue feedback', 'venue feedback', 'venue feedback'], result_hash[:venue_feedback]
-    assert_equal [['Facilitator Bert'], ['Facilitator Ernie'], ['Facilitator Ernie']], result_hash[:who_facilitated]
-    assert_equal({'Facilitator Bert' => 5.0, 'Facilitator Ernie' => 3.0}, result_hash[:how_clearly_presented])
-    assert_equal({'Facilitator Bert' => ['Bert was very strict'], 'Facilitator Ernie' => ['Ernie did not put down the ducky', 'Ernie was disorganized']}, result_hash[:things_facilitator_could_improve])
-
-    result_hash = summarize_workshop_surveys(workshops: workshops, facilitator_name_filter: 'Facilitator Ernie')
-    assert_equal 4.0, result_hash[:how_much_learned]
-    assert_equal ['venue feedback', 'venue feedback', 'venue feedback'], result_hash[:venue_feedback]
-    assert_equal [['Facilitator Bert'], ['Facilitator Ernie'], ['Facilitator Ernie']], result_hash[:who_facilitated]
-    assert_equal 3.0, result_hash[:how_clearly_presented]
-    assert_equal ['Ernie did not put down the ducky', 'Ernie was disorganized'], result_hash[:things_facilitator_could_improve]
-
-    result_hash = summarize_workshop_surveys(workshops: workshops, facilitator_breakdown: false)
-    assert_equal 3.67, result_hash[:how_clearly_presented]
-
-    result_hash = summarize_workshop_surveys(workshops: workshops, facilitator_breakdown: false, include_free_response: false)
-    assert_nil result_hash[:things_facilitator_did_well]
-  end
-
   test 'averaging across multiple surveys' do
-    workshop_1 = create :summer_workshop, num_sessions: 1, num_facilitators: 2, num_completed_surveys: 5
-    workshop_2 = create :summer_workshop, num_sessions: 1, num_facilitators: 3, num_completed_surveys: 10
+    workshop_1 = create :workshop, :teachercon, num_sessions: 1, num_facilitators: 2, num_completed_surveys: 5
+    workshop_2 = create :workshop, :teachercon, num_sessions: 1, num_facilitators: 3, num_completed_surveys: 10
+
+    workshop_1.survey_responses.each do |response|
+      response.update_form_data_hash(
+        {
+          how_happy_after:  'Extremely happy'
+        }
+      )
+      response.save
+    end
 
     workshop_2.survey_responses.each do |response|
       response.update_form_data_hash(
         {
-          how_clearly_presented: {
-            workshop_2.facilitators.first.name => 'Not at all clearly',
-            workshop_2.facilitators.second.name => 'Not at all clearly',
-            workshop_2.facilitators.third.name => 'Not at all clearly'
-          }
+          how_happy_after: 'Not at all happy'
         }
       )
       response.save
     end
```

Comment:
Reason for this test being modified a lot: The test previously used the `how_clearly_presented` question from the Local Summer workshop survey, which was a facilitator-specific numbered/scaled question (i.e. asking about a specific facilitator of the ones that led the workshop, and the question has worded responses that map to scores of 1-5). Since I had to switch to the TeacherCon survey (the only remaining type that `summarize_workshop_surveys` works with), I had to pick a different question but the TeacherCon survey has no facilitator-specific numbered/scaled questions, only non-facilitator-specific ones. So, I arbitrarily picked the `how_happy_after` one which meant the average was now based on total surveys rather than total survey responses about each facilitator.